ssc/cmod_utilityrate5.cpp

```diff
@@ -42,15 +42,15 @@ static var_info vtab_utility_rate5[] = {
 	{ SSC_INPUT, SSC_ARRAY, "gen", "System power generated", "kW", "", "System Output", "*", "", "" },
 	 
 	// input from user as kW and output as kW
-	{ SSC_INOUT, SSC_ARRAY, "load", "Electricity load (year 1)", "kW", "", "Time Series", "", "", "" },
+	{ SSC_INOUT, SSC_ARRAY, "load", "Electricity load (year 1)", "kW", "", "Load", "", "", "" },
```

Comment:
Changing the group name of the outputs will change how they show up in the Data Tables in the UI- is this intentional? I think Paul had done some rearranging of these

Reply:
Maybe we should have Paul do a check for documentation and output grouping in SAM?

Reply:
Oh good point. since it's also an input, I wanted to have an 'input-focused' group name. 

@cpaulgilman Do you have any thoughts on this?